modules/tinymce/src/core/test/ts/browser/content/InsertContentTest.ts

```diff
@@ -352,6 +352,42 @@ describe('browser.tinymce.core.content.InsertContentTest', () => {
     TinyAssertions.assertContent(editor, '<pre> <strong> a </strong> </pre>');
   });
 
+  it('TINY-8860: insertAtCaret - insert pre block on empty p', () => {
```

Comment:
I don't see any test here for inserting into/over a CEF block. Can you please add one?

- [ ] Add a test for pasting over a CEF block